.github/workflows/js-build-and-lint.yml

```diff
@@ -55,14 +59,34 @@ jobs:
           echo "::add-matcher::.github/tsc.json"
           echo "::add-matcher::.github/eslint-stylish.json"
 
-      - name: eslint + fix
-        uses: getsentry/action-eslint-fix@v2
-        if: steps.changes.outputs.frontend == 'true'
-        with:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # Lint entire frontend if this is on main branch
+      - name: eslint
+        if: github.ref == 'refs/heads/master' && steps.changes.outputs.frontend == 'true'
+        run: |
+          # Run relax config on main branch (and stricter config for changed files)
+          yarn lint -c .eslintrc.relax.js
+          yarn lint:css
+
+      # Otherwise if it's not main branch, only lint modified files
+      - name: eslint (changed files only)
+        if: github.ref != 'refs/heads/master' && steps.changes.outputs.frontend == 'true'
+        run: |
+          yarn eslint --fix ${{ steps.changes.outputs.frontend_modified_lintable_files }}
```

Comment:
This is the only occurence of `frontend_modified_lintable_files`. Where does it come from?

Reply:
It comes from the `paths-filter` action, it provides an output of the files: https://github.com/dorny/paths-filter#custom-processing-of-changed-files

Reply:
Oh, I see: `${FILTER_NAME}_files`